era_5g_server/server.py

```diff
@@ -29,9 +29,12 @@
 # Used for Network Application heartbeat.
 MAX_LATENCY = float(os.getenv("NETAPP_MAX_LATENCY", 100))
 
-NETAPP_STATUS_ADDRESS = (
-    MIDDLEWARE_ADDRESS if MIDDLEWARE_ADDRESS.endswith("/status/netapp") else MIDDLEWARE_ADDRESS + "/status/netapp"
-)
+if MIDDLEWARE_ADDRESS.endswith("/status/netapp"):
+    NETAPP_STATUS_ADDRESS = MIDDLEWARE_ADDRESS
+elif MIDDLEWARE_ADDRESS.endswith("/"):
```

Comment:
Just a note - Python has support for joining URLs - it handles slashes, etc. Probably more elegant and bullet-proof than series of conditions :-)

```python
>>> from urllib.parse import urljoin
>>> urljoin('/media/path/', 'js/foo.js')
'/media/path/js/foo.js'
```

Reply:
OK, I didn't know about that